stativus/build.boot

```diff
@@ -0,0 +1,29 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "0.10.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(task-options!
+ pom  {:project     'cljsjs/stativus
+       :version     +version+
+       :description "A Statechart library"
+       :url         "https://github.com/etgryphon/stativus"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"BSD" "http://opensource.org/licenses/BSD-3-Clause"}})
+
+(deftask package []
+  (comp
+   (download :url "https://github.com/etgryphon/stativus/archive/0.10.0.zip"
```

Comment:
Please use `format` (or `str`) to build the URL here using `+lib-version+`, `(format "https://github.com/etgryphon/stativus/archive/%s.zip" +lib-version+)`, this way only the var needs be updated when updating the package and old archive won't get used accidentally.